app/components/cart-items.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
+  fullCart: service('cart')
```

Comment:
nit: having to do this (to not have to use `this.cart.cart`) would be nice to be avoided....
added a suggestion in the service to avoid this